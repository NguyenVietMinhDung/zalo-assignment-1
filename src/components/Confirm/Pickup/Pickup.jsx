import React from 'react';
import { Container, Row, Col } from 'react-grid-system';
import './Pickup.scss';
import Address from './Address';
import Note from './Note';
import Timer from './Timer';

const Pickup = () => (
  <Container>
    <Row className="pickup">
      <Col xs={12}>
        <h3>Tự đến lấy</h3>
      </Col>
      <Col xs={12}>
        <Address />
      </Col>
      <Col xs={12}>
        <Timer />
      </Col>
      <Col xs={12}>
        <Note />
      </Col>
    </Row>
  </Container>
);

export default Pickup;
