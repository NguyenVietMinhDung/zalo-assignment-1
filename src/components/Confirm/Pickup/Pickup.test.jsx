import React from 'react';
import { shallow } from 'enzyme';
import Pickup from './Pickup';

describe('<Pickup />', () => {
  test('renders correctly', () => {
    const component = shallow(<Pickup />);
    expect(component).toMatchSnapshot();
  });
});
