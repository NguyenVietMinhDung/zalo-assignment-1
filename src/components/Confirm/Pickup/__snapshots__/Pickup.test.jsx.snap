// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Pickup /> renders correctly 1`] = `
<Container
  component="div"
  fluid={false}
  lg={false}
  md={false}
  sm={false}
  style={Object {}}
  xl={false}
  xs={false}
  xxl={false}
>
  <Row
    align="normal"
    className="pickup"
    component="div"
    debug={false}
    gutterWidth={null}
    justify="start"
    nogutter={false}
    nowrap={false}
    style={Object {}}
  >
    <Col
      component="div"
      debug={false}
      lg={null}
      md={null}
      offset={Object {}}
      pull={Object {}}
      push={Object {}}
      sm={null}
      style={Object {}}
      width={null}
      xl={null}
      xs={12}
      xxl={null}
    >
      <h3>
        Tự đến lấy
      </h3>
    </Col>
    <Col
      component="div"
      debug={false}
      lg={null}
      md={null}
      offset={Object {}}
      pull={Object {}}
      push={Object {}}
      sm={null}
      style={Object {}}
      width={null}
      xl={null}
      xs={12}
      xxl={null}
    >
      <Address />
    </Col>
    <Col
      component="div"
      debug={false}
      lg={null}
      md={null}
      offset={Object {}}
      pull={Object {}}
      push={Object {}}
      sm={null}
      style={Object {}}
      width={null}
      xl={null}
      xs={12}
      xxl={null}
    >
      <Timer />
    </Col>
    <Col
      component="div"
      debug={false}
      lg={null}
      md={null}
      offset={Object {}}
      pull={Object {}}
      push={Object {}}
      sm={null}
      style={Object {}}
      width={null}
      xl={null}
      xs={12}
      xxl={null}
    >
      <Note />
    </Col>
  </Row>
</Container>
`;
